bindata/v4.0.0/configmap-cabundle-controller/clusterrole.yaml

```diff
@@ -12,19 +12,4 @@ rules:
   - list
   - watch
   - update
-- apiGroups:
-  - ""
-  resources:
-  - configmaps
-  verbs:
-  - get
-  - list
-  - watch
-  - update
-  - create
-- apiGroups:
-  - ""
-  resources:
-  - events
-  verbs:
   - create
```

Comment:
should not need create

Reply:
👍 I'm all for removing what permissions we can. a84d0954